integrators/ordinary_differential_equations.hpp

```diff
@@ -34,9 +34,137 @@ using base::Status;
 using geometry::Instant;
 using numerics::DoublePrecision;
 using quantities::Difference;
+using quantities::Quotient;
 using quantities::Time;
 using quantities::Variation;
 
+// A differential equation of the form y′ = f(y, t).
+// |State| is the type of y.
+template<typename... StateTypes>
```

Comment:
`StateTypes` sounds like Hungarian to me.  Maybe `StateElements`?

Reply:
Done.